akka-actor/src/main/scala/akka/actor/Address.scala

```diff
@@ -65,6 +68,19 @@ final case class Address private (protocol: String, system: String, host: Option
    * `system@host:port`
    */
   def hostPort: String = toString.substring(protocol.length + 3)
+
+      /** INTERNAL API
```

Comment:
weird formatting here

Reply:
Fixed, needed to run fmt

Reply:
Isn't "format on save" in IntelliJ? That must be annoying. I never run sbt scalafmt

Reply:
That became too annoying while adding something new. I may try it again in future.